requirements.txt

```diff
@@ -41,7 +41,7 @@ pytest-mock==1.1
 typing==3.6.4
 
 
-amundsen-common==0.1.3
+amundsen-common==0.1.7rc0
```

Comment:
@feng-tao Seems build is failing due to this one. Could you revert this one as well?